toolkit/scripts/toolchain/build_official_toolchain_rpms.sh

```diff
@@ -316,36 +315,17 @@ build_rpm_in_chroot_no_install python3
 rm -vf $FINISHED_RPM_DIR/python3*debuginfo*.rpm
 chroot_and_install_rpms python3
 
-
-# openjdk needs fontconfig to build, which needs libxml
-build_rpm_in_chroot_no_install libxml2
```

Comment:
Should we also remove `libxml2` packages from the toolchain manifest files?

Reply:
Nevermind, I see now it was simply moved a bit later in the script.

Reply:
Yes, there are a number of packages that depend on libxml2. After seeing build errors I'm going to move it back to this spot.